modules/exploits/multi/http/rails_dynamic_render_code_exec.rb

```diff
@@ -0,0 +1,246 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Ruby on Rails Dynamic Render File Upload Remote Code Execution',
+      'Description'    => %q{
+        This module exploits a remote code execution vulnerability in the explicit render
+        method when leveraging user parameters.
+        This module has been tested across multiple versions of Ruby on Rails.
+        The technique used by this module requires the specified
+        endpoint to be using dynamic render paths, such as the following example:
+
+        def show
+          render params[:id]
+        end
+
+        Also, the vulnerable target will need a POST endpoint for the TempFile upload, this
+        can literally be any endpoint. This module doesnt use the log inclusion method of
+        exploitation due to it not being universal enough. Instead, a new code injection
+        technique was found and used whereby an attacker can upload temporary image files
+        against any POST endpoint and use them for the inclusion attack. Finally, you only
+        get one shot at this if you are testing with the builtin rails server, use caution.
```

Comment:
Normally I would agree an RCE should be an ExcellentRanking, however, I wonder if it should be lowered due to this warning